### DISCUSSION TITLE
Airtable to SQL database, NocoDB roadmap [Jun 25]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2545] [Discord Threads]
> 🙋‍♂️ **cirsten**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### Discord is for live chat. You can raise an issue in github. It will be easier  
#  
> 🙋‍♂️ **cirsten**  
#### Sorry, meant the the "discussion" page for airtable migration on github. There is already a discussion started there, by yourself I think, should I add to that discussion or raise "feature requests" just want to put in the best place    
####    
#### https://github.com/nocodb/nocodb/discussions/2122   
####    
#### "Importing Airtable To NocoDB"  
#  
> 👨‍🦲 **dstala**  
#### <@960663159637688360> You can create a new "Feature Request" under Github issues section. You can do a quick search on Features to see if any similar requests are already in place.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/989796476823683082)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
